pkl-core/src/test/kotlin/org/pkl/core/ErrorColoringTest.kt

```diff
@@ -0,0 +1,51 @@
+package org.pkl.core
+
+import org.assertj.core.api.Assertions.assertThat
+import org.fusesource.jansi.Ansi
+import org.junit.jupiter.api.*
+
+class ErrorColoringTest {
+  companion object {
+    val evaluator by lazy { Evaluator.preconfigured() }
+
+    @AfterAll
+    @JvmStatic
+    fun afterAll() {
+      evaluator.close()
+    }
+  }
+
+  private fun evaluate(program: String, expression: String): Any {
+    return evaluator.evaluateExpression(ModuleSource.text(program), expression)
+  }
+  @BeforeEach
```

Comment:
```suggestion
  }

  @BeforeEach
```

nit (surprised Spotless doesn't balk at this)